### PR TITLE
Create eip-666.md

### PR DESCRIPTION
**Pull request by gcolvin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-666.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+eip: 666
```
Comment (MicahZoltu) [inline EIPS/eip-666.md:2]:
```suggestion
eip: 4887
```
Filename should also be changed to `EIPS/eip-4887.md`

File: EIPS/eip-666.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Eternal.
+
+## Test Cases
+
+[Shut up the words, and seal the book, even to the time of the end.](https://www.kingjamesbibleonline.org/Daniel-Chapter-12/#4)
+
+## Reference Implementation
+
+[The Revelation of St John -- Albrecht Dürer](https://upload.wikimedia.org/wikipedia/commons/c/cf/Durer%2C_apocalisse%2C_03_ventiquattro_vecchi.jpg)
```
Comment (MicahZoltu) [inline EIPS/eip-666.md:40]: External links should not be included.  A reference implementation can be included in `../assets/eip-4887/*`, or the section can be left out if a standalone reference implementation doesn't make sense.

File: EIPS/eip-666.md (hunk 1 above)
Comment (gcolvin) [reply EIPS/eip-666.md:2]: Take it up with St. John ;)

File: EIPS/eip-666.md (hunk 2 above)
Comment (gcolvin) [reply EIPS/eip-666.md:40]: I'll move implementation to ../assets before this is merged, thanks.

Comment (gcolvin) [reply EIPS/eip-666.md:40]: Even better -- how do I get an image to display?

File: EIPS/eip-666.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+eip: 666
+title: The Number of the Beast
+description: I beheld another beast coming up out of the earth; and he had two horns like a lamb, and he spake as a dragon.
+author: Beelzebub (@beelzebub)
+discussions-to: https://hell.com
```
[the remark this answers is not in the document]
Comment (gcolvin) [reply EIPS/eip-666.md:6]: At the author's discretion.  Argue with the author at your own risk ;)

**Pull request closed without merging**